acceptancetests/assess_lxd_space.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+"""Testing spaces and subnets settings for app deployment to lxd container.
+   This is an validation of Bug 1685782:
```

Comment:
Please add a real explanation here. Starting with juju 2.1, spaces must be specified for containers, and juju will pick a default if not specified. This test is to validate that LXD containers don't inherit spaces from the host machine.